src/collections/_documentation/workflow/releases/release-automation/circleci/index.md

```diff
@@ -0,0 +1,74 @@
+
+Before starting, confirm that your Sentry project is properly set up to track commit metadata. The easiest method to do so is to [install a repository integration]({%- link _documentation/workflow/releases/index.md -%}#install-repo-integration). Once that's installed, and you've added your repository, come back to this guide. If you've already installed a repository integration, you're ready to go.
+
+## Create a Sentry Internal Integration
+
+For CircleCI to communicate securely with Sentry, you'll need to create a new internal integration. In Sentry, navigate to **Settings > Developer Settings > New Internal Integration**.
+
+Give your new integration a name (for example, “CircleCI Deploy Integration”) and specify the necessary permissions. In this case, you need Admin access for “Release” and Read access for “Organization”.
+
+For more details about scopes and API endpoints, see the full documentation on [API Permissions]({%- link _documentation/api/permissions.md -%}).
+
+[{% asset releases/circleci/internal-integration-permissions.png alt="View of internal integration permissions." %}]({% asset releases/jenkins/internal-integration-permissions.png @path %})
+
+Click “Save” at the bottom of the page and grab your token, which you'll need this in the next step.
+
+## Setting Environment Variables in CircleCI
+
+Next, you'll need a few environment variables to configure the Sentry CLI:
+
+- `SENTRY_AUTH_TOKEN` - Your internal integration token.
+- `SENTRY_ORG` - Your Sentry organization slug.
+- `SENTRY_PROJECT` - Your Sentry project slug.
+
+To access you internal integration token securely, store it as an [environment variable on your project](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project):
+
+1. In the CircleCI application, go to your project’s settings by clicking the gear icon on the Pipelines page, or the three dots on other pages in the application.
+2. Click on Environment Variables.
+3. Add a new variable by clicking the Add Variable button and enter `SENTRY_AUTH_TOKEN` as the name and your internal integration token as the value.
+
+[{% asset releases/circleci/project-settings-env-var.png alt="View of CircleCI environment variables." %}]({% asset releases/circleci/project-settings-env-var.png @path %})
+
+The other variables do not contain sensitive information, so you'll use the environment key to define them as environment variables in your config.yml file.
+
+## Create Release and Notify Sentry of Deployment
+
+In order to automate your Sentry release management process you'll need to add the `Create release and notify Sentry of deploy` step below after your deploy step, either in a new job or at the end of your existing deploy job:
```

Comment:
Suggest: "To automate" ("in order to" is wordy and doesn't add meaning)

Add comma after the phrase that ends with "management process"